docker-compose/jaeger-docker-compose.yml

```diff
@@ -3,26 +3,30 @@ version: '2'
 services:
     jaeger-collector:
       image: jaegertracing/jaeger-collector
-      command: ["--cassandra.keyspace=jaeger_v1_dc1", "--cassandra.servers=cassandra", "--collector.zipkin.http-port=9411"]
+      entrypoint: ["/bin/bash", "/bin/wait-for-cassandra.sh", "/go/bin/collector-linux", "--cassandra.keyspace=jaeger_v1_dc1", "--cassandra.servers=cassandra", "--collector.zipkin.http-port=9411"]
       ports:
         - "14269"
         - "14268:14268"
         - "14267"
         - "14250"
         - "9411:9411"
+      volumes:
+        - ./scripts/wait-for-cassandra.sh:/bin/wait-for-cassandra.sh
```

Comment:
I wonder how this would work for people reusing this docker-compose file elsewhere, since `scripts/wait-for-cassandra.sh` is not part of the images.

@pavolloffay @jpkrohling do we know who uses this docker-compose file?

Reply:
I was wondering about that as well. I know the clients use this file as part of their crossdock builds. Might have to embed the script into my new `jaegerrtacing/jaeger-service-base` image.